neo4j/internal/router/router_test.go

```diff
@@ -56,7 +57,7 @@ func TestMultithreading(t *testing.T) {
 		n = n.Add(time.Duration(table.TimeToLive) * time.Second * 2)
 		return n
 	}
-	router := New("router", func() []string { return []string{} }, nil, pool, logger, "routerid", &timer)
+	router := New("router", func() []string { return []string{} }, nil, pool, math.MaxInt64, logger, "routerid", &timer)
```

Comment:
Any reasons for/against using `DefaultConnectionLivenessCheckTimeout` here instead?